Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.mm

```diff
@@ -275,7 +253,7 @@
     if (!node)
         return;
 
-    node->applyBackingStore(this, layerContentsType(), properties);
+    node->applyBackingStore(this, properties);
 }
 
 RemoteLayerTreeNode* RemoteLayerTreeHost::nodeForID(std::optional<PlatformLayerIdentifier> layerID) const
```

Comment:
In this file, there's RemoteLayerTreeHost::mapAllIOSurfaceBackingStores which migrates layer contents from CAMachPorts to straight IOSurfaces. Maybe remove that code?

Reply:
Good catch, will remove that too.